packages/codegen-ui-react/lib/__tests__/__snapshots__/studio-ui-codegen-react-forms.test.ts.snap

```diff
@@ -4444,7 +4444,7 @@ export default function PostUpdateForm(props) {
             })
           )?.data?.getPost
         : postModelProp;
-      const linkedComments = record ? record.Comment?.items : [];
+      const linkedComments = record ? record.Comment?.items || [] : [];
```

Comment:
Think this could be simplified as `record?.Comment?.items ?? []`?

Reply:
Yeah ill update it